src/boot/efi/boot.c

```diff
@@ -118,17 +118,20 @@ static BOOLEAN line_edit(
         while (!exit) {
                 EFI_STATUS err;
                 UINT64 key;
-                UINTN i;
 
-                i = len - first;
-                if (i >= x_max-1)
-                        i = x_max-1;
-                CopyMem(print, line + first, i * sizeof(CHAR16));
-                while (clear > 0 && i < x_max-1) {
-                        clear--;
-                        print[i++] = ' ';
+                {
+                        UINTN i;
+
+                        i = len - first;
+                        if (i >= x_max-1)
+                                i = x_max-1;
+                        CopyMem(print, line + first, i * sizeof(CHAR16));
+                        while (clear > 0 && i < x_max-1) {
+                                clear--;
+                                print[i++] = ' ';
+                        }
+                        print[i] = '\0';
```

Comment:
Hmm, such an extra indentation should not be introduced...

Reply:
I used different variable names for the iterators instead of indenting.